onert-micro/luci-interpreter/src/core/RuntimeGraph.h

```diff
@@ -29,42 +29,33 @@ namespace luci_interpreter
 
 class RuntimeModule;
 
-class RuntimeGraph
+class BaseRuntimeGraph
```

Comment:
```suggestion
class IBaseRuntimeGraph
```
since this class has become abstract, I suggest using an "I" prefix for readability

Reply:
Done